front/components/V2/SongCardInfo.tsx

```diff
@@ -0,0 +1,259 @@
+											fontSize: 12,
+											fontWeight: 'bold',
+										}}
+									>
+										{score.score}
+									</Text>
+								</View>
+							))}
+						</View>
+					</View>
+					<PresenceTransition
+						style={{
+							width: '100%',
+							height: '100%',
+							position: 'absolute',
+						}}
+						visible={isHovered}
+						initial={{
+							translateY: 0,
+						}}
+						animate={{
+							translateY: -55,
+						}}
+						onTransitionComplete={() => {
+							if (isHovered) {
+								setIsSlided(true);
+							}
+						}}
+					>
+						<Image
+							source={{ uri: props.song.cover }}
+							style={{
+								position: 'relative',
+								width: CardDims.width,
+								height: CardDims.height,
+								borderRadius: 12,
+							}}
+						/>
+						<View
+							style={{
+								position: 'absolute',
+								width: '100%',
+								height: '100%',
+								backgroundColor: 'rgba(0, 0, 0, 0.75)',
+								justifyContent: 'flex-end',
+								alignItems: 'flex-start',
+								paddingHorizontal: 10,
+								paddingVertical: 7,
+								borderRadius: 12,
+							}}
+						>
+							<View
+								style={{
+									width: '100%',
+									display: 'flex',
+									flexDirection: 'row',
+									alignItems: 'flex-end',
+									justifyContent: 'space-between',
+								}}
+							>
+								<View
+									style={{
+										flexShrink: 1,
+									}}
+								>
+									<Text
+										numberOfLines={2}
+										style={{
+											color: 'white',
+											fontSize: 14,
+											fontWeight: 'bold',
+											marginBottom: 4,
+										}}
+									>
+										{props.song.name}
+									</Text>
+									<Text
+										numberOfLines={1}
+										style={{
+											color: 'white',
+											fontSize: 12,
+											fontWeight: 'normal',
+										}}
+									>
+										{props.song.artistId}
+									</Text>
+								</View>
+								<Ionicons
+									style={{
+										flexShrink: 0,
+									}}
+									name="bookmark-outline"
+									size={17}
+									color="#6075F9"
+								/>
+							</View>
+						</View>
+					</PresenceTransition>
+					{/* icon bu=outon appear in the middle of the card after the first presencetransition is done */}
+					<PresenceTransition
+						style={{
+							width: '100%',
+							height: '100%',
+							position: 'absolute',
+						}}
+						visible={isSlided}
+						initial={{
+							opacity: 0,
+						}}
+						animate={{
+							opacity: 1,
+						}}
+					>
+						<View
+							style={{
+								position: 'absolute',
+								width: '100%',
+								height: '100%',
+								display: 'flex',
+								justifyContent: 'flex-end',
+								alignItems: 'center',
+							}}
+						>
+							<Button
+								onHoverIn={() => {
+									setIsPlayHovered(true);
+								}}
+								onHoverOut={() => {
+									setIsPlayHovered(false);
+								}}
+								borderRadius={100}
+								marginBottom={35}
+								leftIcon={
```

Comment:
This makes the icon to be slightly off-centered to the left. This should  be fixed